docs/source/configuration_file.rst

```diff
@@ -981,6 +981,20 @@ being used for the segmentation task).
                    "Final activation layer. Options: ``sigmoid`` (default), ``relu``(normalized ReLU), or ``softmax``."
                ]
            },
+           "length_2D": {
```

Comment:
I'm opening the question of "How to define the default patch size, ie no patch creation, ie full slice without patching".

For now, from what I understand, if these new params are in the config file, then we do patching, otherwise no patching.

Question: Should we have these parameters in the `default_model`? I was going to say no, and recommend a "modified dict" for this "patchy unet".. but maybe, instead, we could have these params in the default model BUT have default values for "no patching" eg `"length_2D": []` (or similar).

Reply:
> For now, from what I understand (new L1211), if these new params are in the config file, then we do patching, otherwise no patching.

Yes, that's correct, `length_2D` and `stride_2D` are optional in the default_model params, and `length_2D` triggers the patching.

I am not against creating a modified dict for "patchy unet" if we want to keep the "default_model" as simple as possible, or having `"length_2D": []` and `"stride_2D": []` as default values for "no patching".

Reply:
I'm leaning towards this option I reckon:
```
 having "length_2D": [] and "stride_2D": [] as default values for "no patching"
```